### DISCUSSION TITLE
Language Management on Page level

### DISCUSSION THREAD

**Discussion by ajahangard** [oqtane/oqtane.framework#1298] [Ideas]
I Have a recommendation about language management.
I think it is a good feature to be able to set language at page level.
Currently the language is set globally but if we want to have a multi language site using oqtane it is good to be able to set language at page level.
any ideas?

**Comment by sbwalker** [comment]
@ajahangard I am not sure if I understand this use case. Oqtane supports static localization - which is the ability to localize the static text that is part of the application user experience. It does not support content localization - which is the ability to localize the dynamic content of a page into different languages. So if we focus on static localization, there is the ability for a site administrator to specify the languages which are available for a specific site. If they specify multiple languages, users will be able to select their preferred language from the language picker control displayed next to their username at the top right of the page. This allows them to interact with the page in their preferred language.

**Reply by ajahangard** [reply]
@sbwalker . Since we don't have content localization in Oqtane, it is not very useful for a user or administrator to change the language while the content is still in another language. 
What I  am suggesting is that the site administrator should be able to set language for specific pages in the site so that when the user switches to that pages the local specified is represented to the user.
By this method one can have a multi-language web site which some pages are for example in English and some pages are in Persian. so the content localization is possible by adding language specific contents to appropriate pages.

**Reply by iJungleboy** [reply]
BTW: 2sxc 12 for Oqtane should be released on may the 4th... 🌌

We're not quite done with content localization, but IMHO it should be done by Tuesday :) - just as an inspiration.

**Reply by hishamco** [reply]
I can handle content localization, but I think it's not planned yet, as It's mentioned in another PR by @sbwalker, I think it's CMS feature but it's good if we have it in the future releases

**Reply by iJungleboy** [reply]
Note that for anybody quickly needing content localization it's available in 2sxc 12 for Oqtane.
